pkgs/tools/admin/awscli2/default.nix

```diff
@@ -67,6 +67,7 @@ with py.pkgs; buildPythonApplication rec {
   postPatch = ''
     substituteInPlace setup.cfg \
```

Comment:
This looks like a good use case for `pythonRelaxDeps`. Maybe in the next bump on `master`.